src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -287,6 +305,94 @@ const formConfig = {
         }
       }
     },
+    schoolInformation: {
+      title: 'School Information',
+      pages: {
+        schoolInformation: {
+          path: 'school-information',
+          title: 'School Information',
+          uiSchema: {
+            school: {
+              facilityCode: {
+                'ui:title': 'Please click on the button to search for your school.',
+                'ui:field': SchoolSelectField,
+                'ui:required': formData => !_.get('school.view:cannotFindSchool', formData),
```

Comment:
Will review this with UX- might require additional work. The default required behavior doesn't fit perfectly. 
![screen shot 2018-07-31 at 13 44 59](https://user-images.githubusercontent.com/4998130/43483175-0e9b2c2e-94c8-11e8-8e4b-67fd1064f13d.png)